buildpacks/jvm-function-invoker/src/util/logger.rs

```diff
@@ -0,0 +1,86 @@
+use anyhow::anyhow;
+use std::{fmt::Display, io::Write};
+use termcolor::{Color, ColorChoice, ColorSpec, StandardStream, WriteColor};
+
+pub struct Logger {
+    debug: bool,
+}
+
+impl Logger {
```

Comment:
Should we move logging handling into `libcnb.rs` eventually? Whilst parts of logging might be slightly org-specific (at least in terms of style), it doesn't seem so specific that it would be an issue to have in a library intended for use by the rest of the community?

Reply:
IMO, our Heroku/Salesforce specific logging should be be in `libcnb.rs` but rather a dedicated create we publish on crates.io.

Reply:
That works for me; my main preference is to not have to copy-paste this all :-)

Reply:
I'm 👍 on moving this logging logic out, just wasn't sure how we wanted to structure it. We should definitely figure it out for the node function invoker.